app/Http/Controllers/Store/BundleController.php

```diff
@@ -186,39 +186,48 @@ function ($value) {
             isset($inputs['collected_by']) &&
             isset($inputs['collected_on'])
         ) {
-            // Add the date;
-            $bundle->disbursed_at = Carbon::createFromFormat(
-                'Y-m-d',
-                $inputs['collected_on']
-            )->startOfDay()->toDateTimeString();
-
-            try {
-                // Find and add the carer
-                $carer = Carer::findOrFail($inputs['collected_by']);
-                $bundle->collectingCarer()->associate($carer);
-
-                // Find and add the centre
-                $centre = Centre::findOrFail($inputs['collected_at']);
-                $bundle->disbursingCentre()->associate($centre);
-
-                // Add the current user as disbursingUser.
-                $bundle->disbursingUser()->associate(Auth::user());
-
-                // Store it.
-                $bundle->save();
-
-            } catch (\Exception $e) {
-                // Fires if finOrFail() or save() breaks
-                // Log that error by hand
-                Log::error('Bad transaction for ' . __CLASS__ . '@' . __METHOD__ . ' by service user ' . Auth::id());
-                Log::error($e->getTraceAsString());
-                $errors['transaction'] = true;
-            }
 
-            // Return to Index as we've disbursed, and user may want to search
-            $successRoute =  route(
-                'store.registration.index'
-            );
+            // Check there are actual vouchers to disburse, or this is a bit.
+            if ($bundle->vouchers->count() == 0) {
+
```

Comment:
Extra blank lines and double quoted key strings not in keeping with rest of code style.